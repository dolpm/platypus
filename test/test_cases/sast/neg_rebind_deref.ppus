pipe main |> [] |> [] |> unit {
    int x <| 4;
    &int y <| &x;
    mut int z <| 4;
    z <| @y;
    |> ();
}