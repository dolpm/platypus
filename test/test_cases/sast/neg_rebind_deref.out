Fatal error: exception Failure("Can't bind a deref because it is an owned value. Please clone the value before binding it.")
